INSTALL

```diff
@@ -47,13 +47,15 @@ The CMake build system is used. It is recommended to use an out-of-tree build
 directory. The compilation and installation is configured by the following CMake
 variables which you can set on the initial cmake call and alter with ccmake:
 
-CMAKE_INSTALL_PREFIX         = /usr/            # the prefix
-CMAKE_INSTALL_SYSCONF_PREFIX = $(DESTDIR)/etc/  # path to etc directory
+    CMAKE_INSTALL_PREFIX         = /usr/            # the prefix
+    CMAKE_INSTALL_SYSCONF_PREFIX = $(DESTDIR)/etc/  # path to etc directory
 
 Individual paths for special files can be set with the *DIR variables, typically
 relative to CMAKE_INSTALL_PREFIX. If you are building package, you would
 typically build regularly, and install with DESTDIR=./path/to/fakeroot.
 
+    mkdir build && cd build
```

Comment:
Feels a bit redundant to have it this here and in the three other places below

Reply:
Right. But I think it's fine to have this multiple times. I remember that I struggled a lot with these out-of-tree builds because it felt so unfamiliar to manually create a new folder when (only) compiling an existing project.

Reply:
Alright, I'll just leave the PR as is then, thank you for the feedback!